### ISSUE TITLE
bytes.append(double) and bytes.parseDouble() problematic for some numbers

### ISSUE THREAD

**Issue by JerryShea** [OpenHFT/Chronicle-Bytes#85]
bytes.append(6.85202d).toString() results in 6.8520200000000008

In this case we behave differently to Double.toString and Double.parseDouble - see the unit test.

Seems that underlying problem is in BytesInternal.asDouble. Have changed the test to call this more directly.

**Comment by JerryShea** [comment]
```
    @Test
    public void bytesParseDouble_Issue85_Many() {
        double increment = 1e-6d;
        int different = 0;
        int max = 1_000_000;
        for (int i = 0; i< max; i++) {
            String s = String.format("%.6f", i * increment);
            double d = Double.parseDouble(s);
            double d2 = Bytes.from(s).parseDouble();
            if(d != d2) {
                System.out.println(d + " != " + d2);
                ++different;
            }
        }
        System.out.println("Different "+(100.0*different)/max+"%");
    }
```
gives
```
...
...
0.999056 != 0.9990559999999999
0.999639 != 0.9996389999999999
0.999944 != 0.9999439999999999
Different 0.1866%
```

**Comment by peter-lawrey** [comment]
Related to https://github.com/OpenHFT/Chronicle-Wire/issues/110

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.9](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.9), [BOM-2.17.119](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.119)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.9](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.9), [BOM-2.17.196](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.196)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.9](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.9), [BOM-2.17.225](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.225)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.17.9](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.17.9), [BOM-2.17.241](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.241)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
